code/modules/jobs/job_types/security_officer.dm

```diff
@@ -281,6 +281,14 @@ GLOBAL_LIST_INIT(available_depts, list(SEC_DEPT_ENGINEERING, SEC_DEPT_MEDICAL, S
 	id = /obj/item/card/id/syndicate_command/crew_id
 	backpack_contents = list(/obj/item/storage/box/survival/syndie=1)
 
+/datum/outfit/job/security/pirate/pilot
+	name = "First mate(pilot)"
+	uniform = /obj/item/clothing/under/costume/singer/yellow
+	shoes = /obj/item/clothing/shoes/jackboots
+	glasses = /obj/item/clothing/glasses/sunglasses/big
+	suit = /obj/item/clothing/suit/jacket/leather/duster/armoured
+	backpack_contents = list(/obj/item/storage/box/survival/security=1)
```

Comment:
survival security box is already included by the box variable, this will make them spawn with 2

Reply:
ooh, alright. i'll get that fixed